data-collector/src/main/kotlin/com/d3/datacollector/repository/BillingRepository.kt

```diff
@@ -7,14 +7,22 @@ package com.d3.datacollector.repository
 import com.d3.datacollector.model.Billing
 import org.springframework.data.jpa.repository.Query
 import org.springframework.data.repository.CrudRepository
+import java.math.BigDecimal
 import java.util.*
 
 interface BillingRepository : CrudRepository<Billing, Long> {
 
-    @Query("SELECT b FROM Billing b WHERE b.domainName = ?1 and b.asset = ?2 and b.billingType = ?3")
-    fun selectByDomainBillingTypeAndAsset(
+    @Query("SELECT b FROM Billing b " +
+            "WHERE b.domainName = ?1 and b.asset = ?2 and b.billingType = ?3" +
```

Comment:
I would use param names instead of numbers.

Reply:
fixed 😄